topics/sequence-analysis/tutorials/ncbi-blast-against-the-madland/tutorial.md

```diff
@@ -0,0 +1,160 @@
+---
+layout: tutorial_hands_on
+
+title: "NCBI BLAST+ against the MAdLand"
+zenodo_link: 'https://zenodo.org/record/4710649'
+questions:
+- "How to access Galaxy server?"
+- "How can we perform Blast analysis on Galaxy?"
+- "What is MAdLand DB?"
+
+objectives:
+- "Load fasta sequence into Galaxy"
+- "Perform NCBI-Blast+ analysis on Galaxy"
+
+time_estimation: "20m"
+key_points:
+- Blast tool searches a database of sequences for similar sequences to a query sequence.
+- MAdLand is a database of fully sequenced plant and algal genomes, with an emphasis on non-seed plants and streptophyte algae that can be use for sequence similarity search.
+
+contributors:
+- deeptivarshney
+
+---
+
+# Introduction
+
+<!-- This is a comment. -->
+
+MAdLand is a collection of fully sequenced plant and algal genomes, with a focus on non-seed plants and streptophyte algae. For comparison, it includes genomes from fungi, animals, the SAR group, bacteria, and archaea. It is developed and maintained by the [Rensing lab](http://plantco.de). The species are abbreviated by a 5 letter code, which consists of the first three letters of the genus and the first two of the species name, e.g. CHABR for Chara braunii. We add the gene ID to that and additional shortcuts, like whether it is plastome encoded (pt) or transcriptome-based (tr, in cases when no genome is available yet).
+
+> <agenda-title></agenda-title>
+>
+> In this tutorial, we will cover:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+# Access the Galaxy server
+
+> <hands-on-title>Log in or register</hands-on-title>
+>
+> 1. Open your favorite browser (Chrome/Chromium, Safari, or Firefox, but not Internet Explorer/Edge!)
+> 2. Browse to [Galaxy](https://usegalaxy.eu) 
+> 3. Choose *Login or Register* from the navigation bar at the top of the page
+> 4. If you have previously registered an account with this particular instance of Galaxy (user accounts are *not* shared between public servers!), proceed by logging in with your registered *public name*, or email address, and your password.
+>
+>    If you need to create a new account, click on *Register here* instead.
+>
```

Comment:
I don't think this is really necessary in that case, since it is covered by the trainings included in the **Introduction to Galaxy Analyses** section.